core/src/main/scala/filodb.core/metadata/Schemas.scala

```diff
@@ -58,6 +58,7 @@ final case class DataSchema private(name: String,
 final case class PartitionSchema(columns: Seq[Column],
                                  predefinedKeys: Seq[String],
                                  options: DatasetOptions) {
+  val columnIdxLookup = columns.zipWithIndex.map { case(col, idx) => col.name -> Tuple2(col, idx) }.toMap
```

Comment:
``Tuple2(col, idx)`` can be written as ``(col, idx)``

Reply:
IDE is complaining when i was running test cases, just to please IDE I added this.